README.md

```diff
@@ -258,6 +272,20 @@ See [miner documentation](http://daffl.github.com/miner/) for all available tunn
 
 If you plan to use these, be sure to add [electron](https://npmjs.com/electron) and/or [phantomjs-prebuilt](https://npmjs.com/phantomjs-prebuilt) to your list of `devDependencies`, or have them installed globally on your system.
 
+## Chrome Headless
+
```

Comment:
This misses setting launchpad's chromium path so that the chromium instance that puppeteer installs gets executed.

Reply:
e.g.
```bash
testee src/test.html -c testee-config.js
```

```javascript
const puppeteer = require('puppeteer');
process.env.LAUNCHPAD_CHROMIUM = puppeteer.executablePath();

module.exports = {
  browsers: [
    {
      browser: 'chromium',
      args: [
        '--headless',
        '--disable-gpu',
        '--remote-debugging-port=9222'
      ]
    }
  ]
};
```

Reply:
Oh, yuck. I didn't see that. I should document the `LAUNCHPAD_*` env var support in Launchpad and plop @nlundquist's config here. There also may be a better way to specify the executable and if not, it may be worth adding one.